### ISSUE TITLE
WebMAN game keep prompting me to update

### ISSUE THREAD

**Issue by larsenv** [aldostools/webMAN-MOD#1060]
I'm trying to play Geometry Wars on my PS3, but every time I launch it, it prompts me to download then install update data. I tried to do it and now it's looping and making me do it over and over again. I can exit it though, but what am I supposed to do? I put the rap file in the exdata folder and tried installing via apollo with a spare device which is fat32. My USB which I want to use is formatted as ntfs

Also, TV Show King has a package to unlock the game, and I installed it, but it still doesn't seem to acknowledge it's on

Apollo save tool keeps prompting me to update every time I load it. I can load games on the internal hdd, but on the external hdd this problem occur

**Comment by larsenv** [comment]
@aldostools

**Comment by larsenv** [comment]
This issue has been frustrating for me, I asked elsewhere and I couldn't get a valid solution

**Comment by aldostools** [comment]
In Apollo you can uncheck the option Version Update Check in Settings
![image](https://github.com/aldostools/webMAN-MOD/assets/974897/8230b186-5db0-4ff2-8853-e603f428918d)

In the update is being asked by PKG games launch from external storage, try installing them on the internal HDD

**Comment by larsenv** [comment]
I have already confirmed it works if I use the HDD which is internal, but I'm confused because I couldn't find anyone else with the problem. My internal HDD's only 60GB so I want to if possible install to the external HDD, which is in ntfs. Could it be an issue with my game? I used liberator to convert it to iso

**Comment by aldostools** [comment]
PSN liberator may cause issues with some games that use hard coded paths instead of relative paths.

**Comment by larsenv** [comment]
Call of Duty Black Ops II was also giving me the issue. However, games are supposed to update without any kind of looping am I right

**Comment by aldostools** [comment]
Yes, but it depends. It could loop if the updates were installed in wrong order or if the title id changed or if the update cannot write to the storage device (e.g /net0, BD or DVD discs, etc.) among other reasons.

**Comment by larsenv** [comment]
Thanks, I have turned on external USB data and it has significantly helped, only problem is that it's quite slow. I'm surprised that no one's reported this issue before. Was I doing something wrong?

**Comment by aldostools** [comment]
Games in PKG format or JB folders are extremely slow when they execute from external  USB drive.

For USB drives it's recommended to use split ISO for FAT32 or single ISO for NTFS or exFAT (split ISO is also supported).

**Comment by larsenv** [comment]
you can only use external game data on fat32 right

**Comment by aldostools** [comment]
Right. PS3 does not allow other file systems for external devices than FAT32.

**Comment by larsenv** [comment]
Is there a good option to convert the pkg to iso and keep the update embedded?

**Comment by aldostools** [comment]
PSN Liberator is an easy method to convert PKG games to ISO
https://www.psx-place.com/resources/psn-liberator.869

Not all games are compatible. Note: It's flagged as "virus" by Chrome. As far as I know it's a false positive. Anyway use with care.

**Comment by larsenv** [comment]
I was using that but it prompted me to update the game so what can I do, and how can I make sure it splits it for fat32?

**Comment by aldostools** [comment]
Download the PKGs of the game and its updates. Then extract them to a folder in the correct order (first game, then updates in sequential order if there are more than one PKG).

Use PSN Game Folder to create the ISO. It shouldn't ask for update since you're creating the ISO already updated.

You can extract the DLC in the folder too or include them in the PS3_EXTRA folder.

**Comment by larsenv** [comment]
After all this time I figured out I can take the hdd out of the bottom of my phat and then upgrade it, I feel dumb now, would all I need to do is reinstall the packages?

**Comment by aldostools** [comment]
I'm not sure what you're trying to do 🤕

If you reinstall the PKGs the games will  be played from the internal HDD except if you activated external Game Data.

If the PKGs are extracted to a subfolder in /GAMEI of a FAT32 disk the game can be mounted through app_home icon using webMAN MOD. In this format the game folder can be easily updated just copying the files of the game update over the existing files.

To update the files in ISO format you need to extract the ISO and rebuild it again.

**Comment by larsenv** [comment]
I was trying to play games off an external hdd because my internal hdd doesn't have much storage. I figured out that I can open the cover on the bottom of the unit and take out the hard drive and then upgrade it, which is much easier than I believed it would be. I wanted to know if my content will easily transfer over if I upgrade it

**Comment by Masamune3210** [comment]
should, pretty sure the external drive stuff just applies redirects in memory, the game files are left alone so you should be able to just copy them into the correct place on the internal drive and have them show up iirc

**Comment by aldostools** [comment]
> I was trying to play games off an external hdd because my internal hdd doesn't have much storage. I figured out that I can open the cover on the bottom of the unit and take out the hard drive and then upgrade it, which is much easier than I believed it would be. I wanted to know if my content will easily transfer over if I upgrade it

If you upgrade your HDD you need to re-install all the packages in /dev_hdd0/game. It's highly recommended that before upgrade you make a backup of your /dev_hdd0/home directory. In that location are stored the saves, trophies, licenses, messages and other important information for the user.

> should, pretty sure the external drive stuff just applies redirects in memory, the game files are left alone so you should be able to just copy them into the correct place on the internal drive and have them show up iirc

Yes, the files external drive are redirected in memory by the backup managers to /dev_bdvd. When the ISO games are played, the system first searches the content in HDD and the rest of the content in /dev_bdvd.

**Comment by larsenv** [comment]
my last question is if it's possible to install pkg in batch if you have em on ntfs

**Comment by larsenv** [comment]
to the hdd

**Comment by larsenv** [comment]
not on the actual external

**Comment by aldostools** [comment]
No, unless you create an ISO containing all the packages that you want to install in batch.

You can install a batch of packages on FAT32 or remote folder mounted as /dev_bdvd using Install All Packages from side menu of Package Manager > Install Package Files > Standard.

**Comment by larsenv** [comment]
What would you use to create the ISO? Would it install to the HDD? and if you wanna play ps1 or ps2 games would it be possible to put them on internal hdd and does it support eyetoy

**Comment by aldostools** [comment]
To create PS3 ISO from GAMES folder in PS3 use: multiMAN, IRISMAN or ManaGunZ

To dump PS1 or PS2 disc in ISO format from PS3 use: multiMAN. IRISMAN and ManaGunZ can do it too, but I prefer multiMAN.

To create PS3 ISO from folders in PC you can use makeISO included in ps3netsrv or PS3 Tools by Rudy Rastelli.
https://www.psx-place.com/threads/ps3-iso-tools-by-rudi-rastelli.403/

All ISO (ps1, ps2, ps3, psp) can be played from internal HDD or external USB FAT32, NTFS or exFAT.

FAT32 requires the ISO to be split in multiple files due that file system does not support files larger than 4GB.

AFAIK ps2_netemu doesn't support many periferals. I think eyetoy is not supported either.

Not sure if PS3 consoles with PS2 backward compatibility (original fat models) support the eyetoy periferal.

**Comment by larsenv** [comment]
Sorry, how do you copy the PS2 games from an external to an internal?

**Comment by aldostools** [comment]
Use FTP Client or multiMAN File Manager or IRISMAN File Manager.

**Comment by larsenv** [comment]
Alright, I'm kind of confused on how you convert the PKG to an ISO containing all of them though? How would you do it, and how do you install it with WebMAN? Sorry regarding all these question

**Comment by aldostools** [comment]
The compatibility is not good. Anyway here is one tool that allows to conver PKG to ISO easily:
https://www.psx-place.com/threads/psn-liberator-convert-psn-content-to-disc-format.1168/

**Comment by larsenv** [comment]
I could try to use pkgsplitter, but it's telling me that a couple of the packages that are over 4GB are FAT32 ready?

**Comment by aldostools** [comment]
FAT32 does not allow files bigger than 4GB

**Comment by larsenv** [comment]
Yes but it wouldn't allow me to properly split when using a ntfs drive

**Comment by aldostools** [comment]
I recommend you visit my twitter: x.com/aldostools

I have tips and videos that explain all what you're trying to do

**Comment by larsenv** [comment]
I'll take it from here, I quit using twitter entirely and can't even visit it anymore without making myself uncomfortable. I think my pkg splitter was being picky

**Comment by larsenv** [comment]
@aldostools I didn't realize this issue was still open, but I found myself coming back to this.

I was a idiot with this when I tried using webMAN a year ago. I was trying to load games off a external SATA 1TB 2.5" HDD, not realizing that the PS3 uses one itself, so what I did is swap the drive out so I could get space. I figured it out soon after I last responded to this. The weird thing is I could not find anyone who was having this exact problem.

Though 1 question, does webMAN let you copy PS2ISO/PSXISO games from a external HDD to a internal HDD? I'm doing it with IRISMAN as we speak and it's quite slow to start the transfer, but I was having a hard time finding a option for it in the XMB (and it hid the PS2ISO/PSXISO directory)

I was also curious why you were using Twitter for videos instead of a platform like YouTube considering that you have to login in order to view stuff nowadays

**Comment by aldostools** [comment]
Hello,
I'm not a YouTuber. I tweet tips and videos from other people (popular youtubers).

webMAN MOD lets you copy files from external USB drives to internal HDD using the web file manager. However I recommend to use a dedicated file manager file IRISMAN, multiMAN MOD or Simple File Manager MOD.
